projects/gsc/run.py

```diff
@@ -50,7 +47,5 @@
 
     if config is None:
         pass
-    elif "single_instance" in args:
-        imagenet_tune.run_single_instance(config)
     else:
-        imagenet_tune.run(config)
+        vernon.run_with_raytune(config, use_single_instance=args.single_instance)
```

Comment:
`single_instance` is created by `RAY_PARSER` but it is handled outside the `vernon.run_with_raytune`. 
This may lead to inconsistencies when the caller uses `RAY_PARSER` but does call `vernon.run_with_raytune` with the extra parameter.

Reply:
Could you clarify this a bit more? I'm not understanding the implication.

Reply:
I think I get it. You saying that instead of passing `use_single_instance=args.single_instance` to `vernon.run_with_raytune`, I should handle the argument inside `vernon.run_with_raytune` via the config as in 
```
def run(config):

    if config.get("use_single_instance", False):
        return run_single_instance(config)
...
```

Is this correct?

Reply:
yes